### PR TITLE
Edit Terranode.de profile

### PR DESCRIPTION
**Pull request by bitradioUwe**

### CODE REVIEW

**Comment by bitradioUwe** [comment]
Would be great if you can update it. Thanks a lot.
Uwe

**Comment by hanjukim** [comment]
Your website is empty.

File: validators/terravaloper1sz53f8emf76s6gs2u7gdhsr7v0jz4lql3x6nqr/profile.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+    "contact": { // contact info for delegators / outreach
```
Comment (hanjukim) [inline validators/terravaloper1sz53f8emf76s6gs2u7gdhsr7v0jz4lql3x6nqr/profile.json:2]: Comments are not available in JSON.

File: validators/terravaloper1sz53f8emf76s6gs2u7gdhsr7v0jz4lql3x6nqr/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- 0% Commission unti the End of April.
+- commision changes get ammounced 14 days before.
+
+## Contact Us
+Have questions? You can reach us: - Telegram: @DerUwe79
+
+### Website
+http://terranode.de
+http://terra.army
+
```
Comment (hanjukim) [inline validators/terravaloper1sz53f8emf76s6gs2u7gdhsr7v0jz4lql3x6nqr/README.md:22]: websites are empty

File: template/profile.json [hunk 3]
```diff
@@ -1,6 +1,22 @@
-{
-  "contact": {
-    "email": "validator@terra.money",
-    "telegram": "@terravalidator123"
-  }
-}
\ No newline at end of file
+---
```
Comment (hanjukim) [inline template/profile.json:1]: This is not a json file

**Review by hanjukim — CHANGES_REQUESTED** [review]

**Pull request closed without merging**